AutoRest/Generators/CSharp/CSharp/ClientModelExtensions.cs

```diff
@@ -137,8 +137,13 @@ private static bool ShouldValidate(this IType model)
         /// </summary>
         /// <param name="property">The given property documentation to format</param>
         /// <returns></returns>
-        public static string GetFormattedPropertyDocumentation(PropertyTemplateModel property)
+        public static string GetFormattedPropertyDocumentation(Property property)
```

Comment:
Could you make this an extension method like the other methods in this file? That way you can just call property.GetFormattedPropertyDocumentation()